voc/python/ast.py

```diff
@@ -1,4 +1,5 @@
 import ast
+import _ast
 import copy
 import sys
 import traceback
```

Comment:
At column 1: (F401) 'traceback' imported but unused

Reply:
This indeed seems like it can be removed, but has noting to do with this pull request.

Reply:
I removed it in the latest commit.